lib/mdns_lite/configuration.ex

```diff
@@ -37,6 +37,15 @@ defmodule MdnsLite.Configuration do
     GenServer.start_link(__MODULE__, [], name: __MODULE__)
   end
 
+  @spec add_mdns_services(map() | [map()]) :: :ok
+  def add_mdns_services(service) when is_map(service) do
```

Comment:
Currently `MdnsLite.Configuration` isn't a public interface (it has `@moduledoc false`). I think that it either needs to be made a public interface or this `add_mdns_services/1` needs to be `defdelegated` from `MdnsLite`. I'd choose `defdelegate`, but open to comments on that.

Reply:
I’m feeling defdelegate. That’s what it’s for

Reply:
Done